content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -286,15 +286,16 @@ TODO: Test and explain how to use additional non-K8s secrets with an existing se
 
 ## Service Account Token Volume Projection
 
-{{< feature-state for_k8s_version="v1.12" state="beta" >}}
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
 {{< note >}}
-This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
+This TokenRequest and TokenRequestProjection are __GA__ in 1.20 and
 enabled by passing all of the following flags to the API server:
 
 * `--service-account-issuer`
+* `--service-account-key-file`
 * `--service-account-signing-key-file`
-* `--service-account-api-audiences`
+* `--api-audiences`
```

Comment:
GA features are enabled by default. We can omit the note.

Reply:
[Suggested text](https://github.com/kubernetes/website/pull/24823/files#r515684800) added.

Reply:
thanks